### ISSUE TITLE
Create a template processor for the EPANET format

### ISSUE THREAD

**Issue by tasseff** [lanl-ansi/WaterModels.jl#37]
To ensure interoperability with EPANET (e.g., for further testing of design and control decisions), a template processor must be developed that converts solution data from the internal data model to a file that conforms to the EPANET format.

**Comment by ccoffrin** [comment]
a.k.a. `export_epanet`?

**Comment by tasseff** [comment]
Indeed... it looks like [this will be a useful reference on the Julia side](https://github.com/lanl-ansi/PowerModels.jl/blob/master/src/io/matpower.jl) and [this will be a useful reference on the parsing and templating side](https://github.com/USEPA/WNTR/blob/master/wntr/epanet/io.py).
